tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -246,6 +246,7 @@ public void CustomLinkDescriptionPreserve ([Values (AndroidLinkMode.SdkOnly, And
 					},
 					new BuildItem.Source ("LinkerClass.cs") {
 						TextContent = () => @"
+[System.Diagnostics.CodeAnalysis.DynamicDependency (""DynamicDependencyTargetMethod()"", typeof(Library1.LinkerClass))]
 namespace Library1 {
 	public class LinkerClass {
```

Comment:
Can `[DynamicDependency ]` go on a namespace?
* https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.dynamicdependencyattribute

Maybe you can put it on this class's constructor?

Reply:
Ah, forgot to add `[assembly: `